doc/users/event_handling.rst

```diff
@@ -75,6 +75,35 @@ Event name             Class            Description
 'axes_leave_event'     `.LocationEvent` mouse leaves an axes
 ====================== ================ ======================================
 
+.. note::
+   When connecting to 'key_press_event' and 'key_release_event' events,
+   you may encounter inconsistencies between the different user interface
+   toolkits that Matplotlib works with. This is due to inconsistencies/limitations
+   of the user interface toolkit. The following table shows some basic examples of
+   what you may expect to receive as key(s) from the different user interface toolkits,
+   where a comma separates different keys:
+
+   ============== ============================= ============================== ============================= ============================== ==============================
+   Key(s) Pressed WxPython                      Qt                             WebAgg                        Gtk                            Tkinter
+   ============== ============================= ============================== ============================= ============================== ==============================
+   Shift+2        shift, shift+2                shift, "                       shift, "                      shift, "                       shift, "
```

Comment:
What layout are you using? shift+2 is @ on US QWERTY keyboards.

Reply:
A Spanish QWERTY keyboard. Sorry! I forgot to consider that fact.